src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/AmendApplicationAboutToSubmitHandlerTest.java

```diff
@@ -233,5 +313,15 @@ private CallbackRequest doValidCaseDataSetUp(final String path) {
         }
     }
 
+    private FinremCallbackRequest buildCallbackRequest() {
+        return FinremCallbackRequest
+            .builder()
+            .eventType(EventType.AMEND_APP_DETAILS)
+            .caseDetailsBefore(FinremCaseDetails.builder().id(123L).caseType(CONSENTED)
+                .data(new FinremCaseData()).build())
+            .caseDetails(FinremCaseDetails.builder().id(123L).caseType(CONSENTED)
+                .data(new FinremCaseData()).build())
+            .build();
+    }
```

Comment:
Good tests.  Consider adding something to confirm that blank and or empty strings also return no postcode error.

Reply:
[d44db17](https://github.com/hmcts/finrem-case-orchestration-service/pull/1753/commits/d44db173679a3868695fec643f8f2bb8c572870c)